CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -788,7 +788,8 @@ private void RemoveSourceHook()
 
         private void CreateOffscreenBrowserWhenActualSizeChanged()
         {
-            if (browserCreated || System.ComponentModel.DesignerProperties.GetIsInDesignMode(this))
+            var webBrowserInternal = this as IWebBrowserInternal;
```

Comment:
The whole `CreateOffscreenBrowserWhenActualSizeChanged` thing is because of a `CEF` bug, which we really should check to see it's been resolved.